recipes/default.rb

```diff
@@ -106,9 +117,26 @@
   )
 end
 
+# Create zone files if the server is a master server
+if node['bind']['zonetype'] == 'master'
+  all_zones.sort.uniq.each do |zone|
+    template "#{node['bind']['vardir']}/master/db.#{zone}" do
```

Comment:
This might be better suited to a parameterized LWRP or HWRP for simple zone files.  Even then, supporting various types of records may be awkward to use in a LWRP pattern.

Should general zone file management be left up to a wrapper?